yb-voyager/cmd/assessMigrationCommand.go

```diff
@@ -1530,6 +1530,7 @@ func considerQueryForIssueDetection(collectedSchemaList []string) bool {
 }
 
 const (
+	PREVIEW_FEATURES_NOTE = `Some features listed in this report may be supported under a preview flag in the specified target-db-version of YugabyteDB. Please refer to the official <a class="highlight-link" target="_blank" href="https://docs.yugabyte.com/preview/releases/ybdb-releases/">release notes</a> for detailed information and usage guidelines.`
```

Comment:
do we really need to point this out? OR can user's make their decision based on the version sequence.

Reply:
We discussed this in standup, for the preview features in a version it's better to indicate that with a generic note. We are doing this with 2.25.1 because there is  Advisory locks supported in his release as preview feature.